content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,309 @@
+
+## Requirements
+
+### Clusters
+
+This guide requires that you have two Kubernetes clusters with any of the
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
+
+The first cluster is the {{< gloss >}}external control plane{{< /gloss >}} cluster with the Istio `default` profile installed in the `external-istiod` namespace. This istiod serves as the the external control plane for the second cluster.  An ingress gateway is installed in the `istio-system` namespace. The ingress gateway provides mesh sidecars access to the external istiod in the `external-istiod` namespace.
+
+The second cluster is a {{< gloss >}}remote cluster{{< /gloss >}} hosting the mesh. Its Kubernetes API server also provides the configuration for the control plane (istiod) running in the external cluster.
+
+### API Server Access
+
+The API Server in the remote config cluster must be accessible to the external control plane cluster. Many cloud providers make API Servers publicly accessible via network
+load balancers (NLBs). If the API Server is not directly accessible, you will
+have to modify the installation procedure to enable access. For example, the
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) gateway used in
+the multi-network and primary-remote configurations could also be used
+to enable access to the API Server.
+
+## Environment Variables
+
+This guide will refer to two clusters named `external_cluster` and `remote_cluster`. The following environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_EXTERNAL_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the external control plane cluster.
+`CTX_REMOTE_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the remote config cluster.
+`EXTERNAL_ISTIOD_ADDR` | The external host name for the `remote_cluster` to access the external istiod.
+`SSL_SECRET_NAME` | The secret name used to access the ingress gateway on the external control plane cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_EXTERNAL_CLUSTER=external_cluster
+$ export CTX_REMOTE_CLUSTER=remote_cluster
+$ export REMOTE_ISTIOD_ADDR=myexternal-istiod.cloud.com
+$ export SSL_SECRET_NAME = myexternal-istiod-secret
+{{< /text >}}
+
+## Setup
+
+### Setup the external control plane cluster
```

Comment:
This is not the first step, IIUC.  The installation should be:

1. Install the config cluster (in cluster2)
1. Install the primary cluster (in cluster1) that points to the config cluster
1. Install the remote cluster (in cluster2)

Reply:
The steps I have is the simplest steps to set up things.

If you set up config cluster first, you don't know the remote pilot addr yet then you have to go back to re-configure the config cluster.

That is why we setup the first cluster to get the gw and istiod exposed first before anything else.

Also, in external istiod, the config cluster has to be set up first before the external istiod is up.  This is different from primary-remote.

Reply:
You shouldn't need a remote pilot address to set up a config cluster. A config cluster should effectively be it's own profile that just installs CRDs and maybe some secrets ... nothing else.

This flow as-described here is confusing because it's a chicken-or-the-egg situation. It also violates the separation of the mesh operator and admin roles as you described above. You're effectively setting up a remote before you install the control plane.

This needs to be fixed soon ... definitely before 1.9.